plugins/rbac/src/components/CreateRole/RoleForm.tsx

```diff
@@ -126,6 +138,56 @@ export const RoleForm = ({
             );
           }
         }
+
+        if (deleteConditions.length > 0) {
```

Comment:
do we need tests to cover the below scenarios?

Reply:
There are unit tests for the RoleForm component but not form save functionalities such as (create/update) but these are covered by e2e. Maybe we can have a task for adding unit tests for these if needed.

Reply:
I see there is no e2e for testing role deletion. We can have one more task for imrproving e2e as well.

Reply:
in dev mode e2e might not make sense, in showcase we should add  so should be fine for now for this PR